polaris-react/src/styles/shared/_accessibility.scss

```diff
@@ -10,7 +10,7 @@
   // Top position is required to prevent unexpected
   // scrolling with Sheet component
   // https://github.com/Shopify/polaris-react/pull/5208
-  top: 0;
+  top: 0 !important;
```

Comment:
Don't judge me 😓 This is purely for consistency with the rest of the styles here.